Instructions on building docs do not mention need to install numpyro

## Description of your problem

The [instructions on building documentation](https://docs.pymc.io/en/v4.0.1/contributing/build_docs.html) do not mention this, but docs cannot be built without first installing numpyro.

**Steps to reproduce:**

```
$ git clone https://github.com/pymc-devs/pymc.git && cd pymc
$ mkvirtualenv -p ~/.pyenv/versions/3.9.9/bin/python pymc && workon pymc
$ pip install -r requirements-dev.txt
$ make clean
$ make html
<...snip...>
Extension error (sphinx.ext.autosummary):
Handler <function process_generate_options at 0x7f1b2a00b670> for event 'builder-inited' threw an exception (exception: no module named pymc.sampling_jax)
make: *** [Makefile:62: html] Error 2
```

<details><summary>Complete error traceback</summary>

```
sphinx-build docs/source docs/_build -b html
Running Sphinx v5.0.2
WARNING: Invalid configuration value found: 'language = None'. Update your configuration to a valid langauge code. Falling back to 'en' (English).
loading pickled environment... done
[autosummary] generating autosummary for: 404.md, PyMC_and_Aesara.rst, api.rst, api/aesaraf.rst, api/backends.rst, api/data.rst, api/distributions.rst, api/distributions/censored.rst, api/distributions/continuous.rst, api/distributions/discrete.rst, ..., learn/consulting.md, learn/core_notebooks/GLM_linear.ipynb, learn/core_notebooks/dimensionality.ipynb, learn/core_notebooks/index.md, learn/core_notebooks/model_comparison.ipynb, learn/core_notebooks/posterior_predictive.ipynb, learn/core_notebooks/pymc_aesara.ipynb, learn/core_notebooks/pymc_overview.ipynb, learn/usage_overview.rst, learn/videos_and_podcasts.md
WARNING: [autosummary] failed to import pymc.AR1.
Possible hints:
* AttributeError: module 'pymc' has no attribute 'AR1'
* ModuleNotFoundError: No module named 'pymc.AR1'
* ImportError:
WARNING: [autosummary] failed to import pymc.EulerMaruyama.
Possible hints:
* ModuleNotFoundError: No module named 'pymc.EulerMaruyama'
* AttributeError: module 'pymc' has no attribute 'EulerMaruyama'
* ImportError:
WARNING: [autosummary] failed to import pymc.KullbackLiebler.
Possible hints:
* ModuleNotFoundError: No module named 'pymc.KullbackLiebler'
* ImportError:
* AttributeError: module 'pymc' has no attribute 'KullbackLiebler'
WARNING: [autosummary] failed to import pymc.NFVI.
Possible hints:
* ModuleNotFoundError: No module named 'pymc.NFVI'
* ImportError:
* AttributeError: module 'pymc' has no attribute 'NFVI'
WARNING: [autosummary] failed to import pymc.NormalizingFlow.
Possible hints:
* ModuleNotFoundError: No module named 'pymc.NormalizingFlow'
* ImportError:
* AttributeError: module 'pymc' has no attribute 'NormalizingFlow'
WARNING: [autosummary] failed to import pymc.SimulatorRV.
Possible hints:
* AttributeError: module 'pymc' has no attribute 'SimulatorRV'
* ImportError:
* ModuleNotFoundError: No module named 'pymc.SimulatorRV'
WARNING: [autosummary] failed to import pymc.compile_pymc.
Possible hints:
* ImportError:
* AttributeError: module 'pymc' has no attribute 'compile_pymc'
* ModuleNotFoundError: No module named 'pymc.compile_pymc'
WARNING: [autosummary] failed to import pymc.pandas_to_array.
Possible hints:
* ImportError:
* AttributeError: module 'pymc' has no attribute 'pandas_to_array'
* ModuleNotFoundError: No module named 'pymc.pandas_to_array'

Extension error (sphinx.ext.autosummary):
Handler <function process_generate_options at 0x7f1b2a00b670> for event 'builder-inited' threw an exception (exception: no module named pymc.sampling_jax)
make: *** [Makefile:62: html] Error 2
```

</details>

Following the install step suggested in the [installation doc](https://github.com/pymc-devs/pymc/blob/v4.0.1/docs/source/installation.md#jax-sampling) fixes the problem:

```
$ pip install numpyro
$ make clean
$ make html
sphinx-build docs/source docs/_build -b html
Running Sphinx v5.0.2
<...snip...>
build succeeded, 64 warnings.

The HTML pages are in docs/_build.

Build finished. The HTML pages are in docs/_build/html.
```

## Versions and main components

* PyMC/PyMC3 Version: `v4.0.1`
* Aesara/Theano Version: `aesara==2.7.3`
* Python Version: `3.9.9`
* Operating system: Linux (Ubuntu 20.04)
* How did you install PyMC/PyMC3: `pip`

--- a/docs/source/contributing/build_docs.md
+++ b/docs/source/contributing/build_docs.md
@@ -4,6 +4,7 @@ To build the docs, run these commands at pymc repository root:
 
 ```bash
 $ pip install -r requirements-dev.txt  # Make sure the dev requirements are installed
+$ pip install numpyro  # Make sure `sampling_jax` docs can be built
 $ make clean  # clean built docs from previous runs and intermediate outputs
 $ make html   # Build docs
 $ python -m http.server --directory docs/_build/  # Render docs

